### PR TITLE
add wmbot logging

### PR DESCRIPTION
**Pull request by dragonpox**
--add option to !identify #playertag for wmbot support.  (Wmbot must register the bot to be a member of your clan)
--add styles 7 and 8 (style 7 uses wmbot prefix !wm.   style 8 uses wm)

### CODE REVIEW

**Comment by spAnser** [comment]
Will take a look at this when I get a chance. Can you send me an invite to a server where this is working / tested with wmbot? I don't personally use it.

First thought though is can you reverse the emoji part so if I merge it doesn't mess with the original IDs. While the IDs probably don't work for anyone else. I do run a direct clone of this repo for a few hundred clans.

**Comment by dragonpox** [comment]
Sure.   I'll fix that and will DM you an invite shortly

File: index.js [hunk 1]
```diff
@@ -344,6 +344,32 @@ global.discordAttackMessage = (warId, WarData, clanTag, opponentTag, attackData,
     .addField(WarData.stats.clan.name, WarData.stats.clan.tag, true)
     .addField('\u200b', '\u200b', true)
     .addField(WarData.stats.opponent.name, WarData.stats.opponent.tag, true)
+  } else if (style === 7) {
```
Comment (dragonpox) [inline index.js:347]: !wm prefix for logging commands

File: index.js [hunk 2]
```diff
@@ -344,6 +344,32 @@ global.discordAttackMessage = (warId, WarData, clanTag, opponentTag, attackData,
...
+    	text += opponentPlayer.mapPosition + ' '
+    	text += attackData.stars + ' ' + attackData.destructionPercentage
+    	}
+    	else if (attackData.who === 'clan'){
+    	text = ''
+    	text += '!wm ' + clanPlayer.mapPosition + 'a'
+    	text += opponentPlayer.mapPosition + ' '
+    	text += attackData.stars + ' ' + attackData.destructionPercentage
+    	}
+  } else if (style === 8) {
```
Comment (dragonpox) [inline index.js:360]: alternative "wm" prefix for logging commands

File: index.js [hunk 3]
```diff
@@ -742,10 +768,11 @@ DiscordClient.on('message', message => {
...
       helpMessage += '5. `' + prefix + 'playerstats #PLAYERTAG` Display player stats for any player tag provided.\n'
-      helpMessage += '6. `' + prefix + 'style [1-6](+)` Choose a style to use for war attacks in this channel. Requires a number to select style type, optionally append a `+` if you want war stats included in every message.\n'
+      helpMessage += '6. `' + prefix + 'style [1-8](+)` Choose a style to use for war attacks in this channel. Requires a number to select style type, optionally append a `+` if you want war stats included in every message.\n'
       helpMessage += '7. `' + prefix + 'filter all,attacks,defenses,none` Not yet implemented\n'
       // helpMessage += '7. `' + prefix + 'filter all,attacks,defenses,none` Filter which attacks show up in the channel.\n'
-      helpMessage += '8. `' + prefix + 'info` Display bot information.'
+      helpMessage += '8. `' + prefix + 'info` Display bot information.\n'
+      helpMessage += '9. `' + prefix + 'identify` bot as clan member (!identify #aaaaaaa).'
```
Comment (dragonpox) [inline index.js:775]: Identification is required to allow wmbot to recognize that the announcer is a clanmember authorized to make commands

**Pull request merged**